contain-intrinsic-* missing properties (#21110)
mdn/content: files/en-us/web/css/contain-intrinsic-height/index.md

---
title: contain-intrinsic-height
slug: Web/CSS/contain-intrinsic-height
tags:
  - CSS
  - CSS Property
  - Reference
  - experimental
  - contain-intrinsic-height
browser-compat: css.properties.contain-intrinsic-height
---

{{CSSRef}} {{SeeCompatTable}}

> The **`contain-intrinsic-length`** [CSS](/en-US/docs/Web/CSS) property sets the height of an element that will be used for layout when it is subject to [size containment](/en-US/docs/Web/CSS/CSS_Containment#size_containment).

Reviewer: ```suggestion
The **`contain-intrinsic-length`** [CSS](/en-US/docs/Web/CSS) property sets the height of an element that a browser can use for layout when the element is subject to [size containment](/en-US/docs/Web/CSS/CSS_Containment#size_containment).
```
